source/administration/changelog.md

```diff
@@ -6,7 +6,10 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.35 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
-**Release Day: 2021-05-16**
+- **v5.35.1, release day TBD**
+  - Investigating an issue where 5.35.0 migration is failing on MySQL installations with an "invalid connection" error. [MM-35767](https://mattermost.atlassian.net/browse/MM-35767)
```

Comment:
Probably needs more details on what the exact impact is.

Reply:
So, there's an issue with the `readTimeout` parameter in `SqlSettings.DataSource` (default is 30s) . This should be either removed or increased to a high enough value to allow for the migration to happen.

Reply:
Made edits